components/Hamburger/Hamburger.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import HamburgerIcon from '@material-ui/icons/Menu';
+import CloseIcon from '@material-ui/icons/Close';
+
+import styled from 'styled-components';
+import { colors } from '../shared/theme';
+
+const getColors = ({ inverted }) => {
+  const weight = !inverted ? 0 : 900;
+
+  return colors.neutral[weight];
+};
+
+const HamburgerWrapper = styled.div`
+  display: inline-block;
+  position: relative;
+  width: 24px;
+  height: 24px;
+`;
+
+const NotificationIcon = styled.span`
```

Comment:
What you think about change the notification border when the `inverted` prop is false ?

![image](https://user-images.githubusercontent.com/16689908/68126660-04a5e600-fef3-11e9-9ca9-8995fdd64220.png)